### PR TITLE
Add more test cases

### PR DESCRIPTION
**Pull request by gkoz**
The tests are currently failing because of parser bug

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
I work on your branch locally to try to fix it.

File: tests/tests.rs [hunk 1]
```diff
@@ -65,14 +65,206 @@ fn get_basic_cmt(file: &str) -> String {
...
+use Container;
+use Widget;
+use Window;
+use ffi;
+use glib::object::Downcast;
+use glib::object::IsA;
+use glib::translate::*;
+
+glib_wrapper! {
+/// Dialog boxes are a convenient way to prompt the user for a small amount
```
Comment (GuillaumeGomez) [inline tests/tests.rs]: The indent is missing here. The test will still fail because it'll regenerate with an indent of four spaces.

Comment (gkoz) [inline tests/tests.rs]: Oh, I'll fix this then

**Comment by GuillaumeGomez** [comment]
Thanks for your great work. I merge!

**Pull request merged**